usmqe_tests/conftest.py

```diff
@@ -29,6 +29,7 @@ def logger_testcase(request):
     """
     Mark start and end of a test case using usmqe logger.
     """
+    print()
```

Comment:
Because of a new line on std output. Easiest way how to make it happen to look it much better than without.